app/models.py

```
@@ -781,6 +799,25 @@ def smtp_password(self, value: Optional[str]):
+    @property
+    def oidc_client_secret(self) -> Optional[str]:
+        """Decrypt and return OIDC client secret."""
+        if self.oidc_client_secret_encrypted:
+            try:
+                return encryption_manager.decrypt(self.oidc_client_secret_encrypted)
+            except InvalidToken:
+                return self.oidc_client_secret_encrypted
+        return None
+
+    @oidc_client_secret.setter
+    def oidc_client_secret(self, value: Optional[str]):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `oidc_client_secret`

Add return type annotation: `None`

(ANN202)